src/jit/compiler.hpp

```diff
@@ -1289,6 +1289,10 @@ inline void GenTree::SetOper(genTreeOps oper, ValueNumberUpdate vnUpdate)
 
     assert(GenTree::s_gtNodeSizes[oper] == TREE_NODE_SZ_SMALL || (gtDebugFlags & GTF_DEBUG_NODE_LARGE));
 
+#if NODEBASH_STATS
+    RecordOperBashing(OperGet(), oper);
```

Comment:
don't actually need this `#if` the way you've defined RecordOperBashing (and you don't have it in CopyFrom, below)